sim/battle-actions.ts

```diff
@@ -1609,6 +1609,17 @@ export class BattleActions {
 			basePower = 0;
 		}
 
+		if (
+			basePower < 60 && source.getTypes(true).includes(move.type) && source.terastallized && move.priority <= 0 &&
```

Comment:
`source.getTypes(true)` will always return `source.terastallized` if it's set.

Reply:
I'm unsure what you're saying here

Reply:
I don't know what the correct condition is, but `source.getTypes(true).includes(move.type) && source.terastallized` is the same as `move.type === source.terastallized`.

Reply:
Yeah, but the current implementation is used in other places in the sim currently so I wanna keep consistent w/ that

Reply:
I used the new form in #9138 i like it better

Reply:
I couldn't find any other place using `source.getTypes(true)`. I did find one using `source.getTypes(false, true)` but obviously that's completely different.